### PR TITLE
validation: endpoint changed to POST.

### PR DESCRIPTION
**Pull request by oguzdemirbasci**
Ref: https://github.com/inspirehep/inspirehep/issues/1474

### CODE REVIEW

File: inspirehep/modules/editor/api.py [hunk 1]
```diff
@@ -254,21 +254,23 @@ def get_workflow_and_schema(workflow_id):
...
+
+        workflow_db.extra_data['validation_errors'] = errors
+        workflow_db.save()
         db.session.commit()
 
-        return jsonify(workflow.extra_data), 400
+        return jsonify(errors), 400
 
-    return jsonify(workflow.extra_data)
+    return jsonify({'result': 'success'})
```
Comment (drjova) [inline inspirehep/modules/editor/api.py:276]: Why do we need success? isn't the status code enough? Is it expected by record-editor?

Comment (oguzdemirbasci) [reply inspirehep/modules/editor/api.py:276]: Client in record-editor throws json conversion error when only status code returned.

Comment (drjova) [reply inspirehep/modules/editor/api.py:276]: Is the editor expects `result`?

Comment (oguzdemirbasci) [reply inspirehep/modules/editor/api.py:276]: No, it should be a valid JSON format. So I just write it like this. It throws an error when `{'success'}` returned.

Comment (drjova) [reply inspirehep/modules/editor/api.py:276]: `"Success"` is a valid json

**Review by drjova — APPROVED** [review]

**Pull request merged**